config/helmet-csp.js

```diff
@@ -32,14 +30,14 @@ const CSP = {
         'data:',
         'www.google-analytics.com',
         'bootswatch.com',
+        'syndication.twitter.com',
         'stats.g.doubleclick.net',
         'ad.doubleclick.net',
         '*.convertro.com',
         '*.c3tag.com',
         '*.2mdn.net',
         'launchbit.com',
-        'www.launchbit.com',
-        'www.ziprecruiter.com'
+        'www.launchbit.com'
```

Comment:
We should check if the other entries are all needed, but I don't have the time to do it now.

Reply:
Lets do things in small and quick steps. As we remove more pages we can review this and update it too.
I think this PR is ready and after that develop can be merged into production.

Reply:
Sure, just wanted to note the issue :P